src/profiling/profiler.cpp

```diff
@@ -51,9 +47,9 @@ namespace gpu {
 }
 #endif
 
-/////////////////////////////////////////////////////////
-// profiler_node
-/////////////////////////////////////////////////////////
+//
+// profiler_node implementation
```

Comment:
it is the implementation of the `profiler_node`. What more is their to say?

The header file where the type interface is defined is the appropriate spot for documentation (not that there is much). The `profiler_node` is actually a low level, strictly back end thing that doesn't need full API-style documentation.